src/applications/edu-benefits/1995/containers/StemEligibilityView.jsx

```diff
@@ -0,0 +1,246 @@
+import React from 'react';
+import _ from 'lodash';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+import { isChapter33 } from '../helpers';
+
+export class StemEligibilityView extends React.Component {
+  onChange = property => {
+    this.props.onChange({
+      ...this.props.formData,
+      ...property,
+    });
+  };
+
+  icon = indication => (indication ? 'fa fa-check' : 'fa fa-times');
+
+  iconColor = indication =>
+    indication ? 'vads-u-color--green' : 'vads-u-color--gray-medium';
+
+  renderChecks = () => {
+    const {
+      isEdithNourseRogersScholarship,
+      benefit,
+      exhaustionOfBenefits,
+      exhaustionOfBenefitsAfterPursuingTeachingCert,
+      isEnrolledStem,
+      isPursuingTeachingCert,
+    } = this.props;
+
+    const isEdithNourseRogersScholarshipCheck =
+      isEdithNourseRogersScholarship && isChapter33({ benefit });
+    const isEdithNourseRogersScholarshipQuestion =
+      isEdithNourseRogersScholarship && benefit === undefined;
+    const exhaustionOfBenefitsCheck =
+      exhaustionOfBenefits || exhaustionOfBenefitsAfterPursuingTeachingCert;
+    const isEnrolledStemCheck = isEnrolledStem || isPursuingTeachingCert;
+
+    const recipientIcon = isEdithNourseRogersScholarshipQuestion
+      ? 'fa fa-question'
+      : this.icon(isEdithNourseRogersScholarshipCheck);
+
+    const isEdithNourseRogersScholarshipClasses = classNames(
+      recipientIcon,
+      this.iconColor(isEdithNourseRogersScholarshipCheck),
+    );
+    const exhaustionOfBenefitsClasses = classNames(
+      this.icon(exhaustionOfBenefitsCheck),
+      this.iconColor(exhaustionOfBenefitsCheck),
+    );
+    const isEnrolledStemClasses = classNames(
+      this.icon(isEnrolledStemCheck),
+      this.iconColor(isEnrolledStemCheck),
+    );
+
+    return (
+      <div>
+        <p>
+          <span className="vads-u-font-family--serif heading-level-4">
+            Based on you responses, it appears you're not eligible.
+          </span>
+          <br />
+          <br />
+          <b>Your responses:</b>
+        </p>
+        <ul className="fa-ul stem-eligibility-ul">
```

Comment:
`fa-ul` applies the `list-style-type: none` property however this property is overwritten on the Review Page requiring a specific class defined in `src/applications/edu-benefits/sass/edu-benefits.scss` so that icons and bullets both do not appear on the Review Page

I followed https://fontawesome.com/how-to-use/on-the-web/styling/icons-in-a-list to use icons in a list

Reply:
Added `role="list` along with an eslint disable for the file